src/components/form/textInput/index.js

```diff
@@ -1,5 +1,60 @@
 import { useState } from "react";
 
+<<<<<<< HEAD
```

Comment:
You'll need to remove this. Did you run & test the app before resubmitting for review?